File: URI-cpp/1012.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{
    double a,b,c,pi = 3.14159,tri,cir,tra,qua,ret;

    cin >> a >> b >> c;

    tri = (a*c)/2;
    cir = pi*pow(c,2);
    tra = ((a+b)*c)/2;
    qua = pow(b,2);
    ret = a*b;

    cout << "TRIANGULO: " << fixed << setprecision(3) << tri << endl;
    cout << "CIRCULO: " << fixed << setprecision(3) << cir << endl;
    cout << "TRAPEZIO: " << fixed << setprecision(3) << tra << endl;
    cout << "QUADRADO: " << fixed << setprecision(3) << qua << endl;
    cout << "RETANGULO: " << fixed << setprecision(3) << ret << endl;

    return 0;
}

File: URI-lua/1012.lua
a,b,c = io.read("*n","*n","*n")
pi = 3.14159
formas = {"TRIANGULO","CIRCULO","TRAPEZIO","QUADRADO","RETANGULO"}

for i in pairs(formas) do
  if formas[i] == "TRIANGULO" then
    io.write(string.format("%s: %.3f\n",formas[i],(a*c)/2))
  elseif formas[i] == "CIRCULO" then
    io.write(string.format("%s: %.3f\n",formas[i],pi*math.pow(c,2)))
  elseif formas[i] == "TRAPEZIO" then
    io.write(string.format("%s: %.3f\n",formas[i],((a+b)*c)/2))
  elseif formas[i] == "QUADRADO" then
    io.write(string.format("%s: %.3f\n",formas[i],math.pow(b,2)))
  elseif formas[i] == "RETANGULO" then
    io.write(string.format("%s: %.3f\n",formas[i],a*b))
  end
end